### PR TITLE
[CDAP-11355] Fixes styling in Wrangler modal to be consistent with that of DataPrep

### PR DESCRIPTION
**Pull request by tonybach**
JIRA: https://issues.cask.co/browse/CDAP-11355

### CODE REVIEW

File: cdap-ui/app/directives/widget-container/widget-wrangler-directives/wrangler-modal.less [hunk 1]
```diff
@@ -14,21 +14,77 @@
...
+              display: inline-block;
+              padding: 3px;
+              font-weight: bold;
+              line-height: 1;
+              color: #ffffff;
+              text-align: center;
+              white-space: nowrap;
+              border-radius: 4px;
+              &.tag-success {
+                background-color: #5cb85c;
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/widget-container/widget-wrangler-directives/wrangler-modal.less]: Can we extract this color too?

**Review by ajainarayanan — COMMENTED** [review]
Just a minor comment. Otherwise code 👍 LGTM

**Comment by tonybach** [comment]
Compared the screens with Lea (on different browsers), and we found some more discrepancies. Will add a commit to fix these.

**Comment by tonybach** [comment]
**EDIT**: These have been fixed in the latest commit.

Lea found these issues while testing the Wrangler modal in Firefox:
- Browse button in the workspace modal looks truncated:

screen shot 2017-05-01 at 7 32 29 pm

- "Inferred Type" table has different background color (should be same as surrounding):

screen shot 2017-05-01 at 6 54 08 pm

- Filter: Keep rows | Remove rows text look small, and the textarea for Custom condition can be resized without bounds:
screen shot 2017-05-01 at 6 54 41 pm

- In Extract Fields using Patterns modal, the dropdown doesn't show full height, and the modal shows vertical scroll:
screen shot 2017-05-01 at 6 54 56 pm

- Custom fields in Parse as CSV/Simple Data & Format Date Time are not auto-focused.
screen shot 2017-05-01 at 6 56 13 pm

- Extract Fields > Using Delimiters is not clickable.

**Comment by ajainarayanan** [comment]
@tonybach Only noticed one thing, While opening edit workspace from within studio it opens up two this modal,
screen shot 2017-05-02 at 11 05 07 am


Otherwise Code 👍 LGTM

**Comment by ajainarayanan** [comment]
👍 Code LGTM

**Comment by leacd** [comment]
LGTM

**Comment by tonybach** [comment]
Build: https://builds.cask.co/browse/CDAP-DRC6408-6

**Pull request merged**